pkg/controller/sparkapplication/controller_another_test.go

```diff
@@ -0,0 +1,224 @@
+package sparkapplication
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	kubeclientfake "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/record"
+
+	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io/v1beta2"
+	crdclientfake "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/clientset/versioned/fake"
+	crdinformers "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/informers/externalversions"
+	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/util"
+)
+
+// separate this test with the original `controller_test.go` to simplify rebasing process in the future
+
+// newAnotherFakeController is a copy of the function from the original controller_test
+// except we don't enable the UIService (Behavior from ofas spark-operator)
+func newAnotherFakeController(app *v1beta2.SparkApplication, pods ...*apiv1.Pod) (*Controller, *record.FakeRecorder) {
+	crdclientfake.AddToScheme(scheme.Scheme)
```

Comment:
my IDE is giving me warning here and few more places in this test file about unhandled

Reply:
is it for code like ? 
```
	err = ctrl.syncSparkApplication(fmt.Sprintf("%s/%s", app.Namespace, app.Name))
	assert.Nil(t, err)

```

Reply:
that's related to a test file, the test is a duplicate from the original repo.
I will close this review since I don't think it's critical to fix these warn